### PR TITLE
Add functions for POST APIs

### PR DESCRIPTION
**Pull request by 1000ch**

### CODE REVIEW

**Comment by lewisf** [comment]
is there anything i can do to help with fixing the tests up and merging this in?

**Comment by lewisf** [comment]
@andrewdelprete i ended up forking one internally -- but happy to help out with one if you fork it

**Comment by 1000ch** [comment]
@lewisf @andrewdelprete 
Sorry for delay, I can add methods for POST APIs, but cannot test continuously. So I didn't merge this PR. How do you think? Should I remove tests for POST APIs to add methods?

File: index.js [hunk 1]
```diff
@@ -84,6 +105,25 @@ module.exports = class SpeedCurve {
...
+   */
+  addDeploy(siteId, note, detail) {
+    let options = Object.assign({}, this.options, {
+      body : {
+        site_id : siteId,
+        note    : note,
+        detail  : detail
+      }
+    });
+    return request.post(`${API_ENDPOINT}/notes`, options);
```
Comment (andrewdelprete) [inline index.js]: should be /deploys instead of /notes here

**Review by andrewdelprete — CHANGES_REQUESTED** [review]

**Comment by andrewdelprete** [comment]
Hey @1000ch, yeah I'm okay with it personally! It's such a simple wrapper. I've just copied the class to my project but would love to just be able to npm install.

**Comment by andrewdelprete** [comment]
Thanks for this! @1000ch

**Pull request merged**